tests/commands/test_validate.py

```diff
@@ -0,0 +1,85 @@
+# these tests can be run from the command line via
+# python manage.py test tests.commands.test_validate --settings="tests.test_settings"
+
+from io import StringIO
+
+from django.core.management import call_command
+
+from arches.app.const import IntegrityCheck
+from arches.app.models.graph import Graph
+from arches.app.models.models import (
+    DDataType,
+    Node,
+    NodeGroup,
+)
+from tests.base_test import ArchesTestCase
+
+
+class ValidateTests(ArchesTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        """This will probably be moved to a helper somewhere importable
+        by arches applications, but just getting started."""
```

Comment:
Copied this over from arches-querysets, we'll probably keep iterating there and when it's adequate move it over to core arches somewhere importable, but meantime I found it useful here.

cc/ @whatisgalen re our sidebar in Belfast